virttest/shared/cfg/base.cfg

```diff
@@ -12,6 +12,9 @@ qemu_io_binary = qemu-io
 #enable_kvm = yes
 # Explicitly pass -no-kvm to qemu (default no)
 #disable_kvm = no
+# For enabling an accelerator explicitly (default ''). This option has
+# higher priority than `enable_kvm` or `disable_kvm`.
+#vm_accelerator =
```

Comment:
This is used as a parameter for `-machines`, so prefer `machine_accel`. 
There is another `accel` device:
```
       -accel name[,prop=value[,...]]
              This is used to enable an accelerator. Depending on the target architecture, kvm, xen, hax, hvf, whpx or
              tcg can be available. By default, tcg is used. If there is more than one accelerator specified, the next
              one is used if the previous one fails to initialize.
```

Reply:
As we discussed, use `vm_accelerator` for high leave meaning, while implement with machine type parameter now.